WalletWasabi.Gui/Rpc/WasabiJsonRpcService.cs

```diff
@@ -121,18 +121,27 @@ public object GetStatus()
 		}
 
 		[JsonRpcMethod("build")]
-		public string BuildTransaction(PaymentInfo[] payments, OutPoint[] coins, int feeTarget, string password = null)
+		public string BuildTransaction(PaymentInfo[] payments, OutPoint[] coins, int feeTarget = 0, decimal feeRate = 0, string password = null)
 		{
 			Guard.NotNull(nameof(payments), payments);
 			Guard.NotNull(nameof(coins), coins);
-			Guard.InRangeAndNotNull(nameof(feeTarget), feeTarget, 2, Constants.SevenDaysConfirmationTarget);
 			password = Guard.Correct(password);
 
+			FeeStrategy feeStrategy;
+
+			if (feeTarget != 0) {
+				Guard.InRangeAndNotNull(nameof(feeTarget), feeTarget, 2, Constants.SevenDaysConfirmationTarget);
+				feeStrategy = FeeStrategy.CreateFromConfirmationTarget(feeTarget);
+			} else if (feeRate != 0) {
+				feeStrategy = FeeStrategy.CreateFromFeeRate(new FeeRate(feeRate));
+			} else {
+				throw new Exception($"{nameof(feeTarget)} or {nameof(feeRate)} must be set");
+			}
```

Comment:
Please remove java style braces:
```suggestion
			if (feeTarget != 0)
			{
				Guard.InRangeAndNotNull(nameof(feeTarget), feeTarget, 2, Constants.SevenDaysConfirmationTarget);
				feeStrategy = FeeStrategy.CreateFromConfirmationTarget(feeTarget);
			}
			else if (feeRate != 0)
			{
				feeStrategy = FeeStrategy.CreateFromFeeRate(new FeeRate(feeRate));
			}
			else
			{
				throw new Exception($"{nameof(feeTarget)} or {nameof(feeRate)} must be set.");
			}
```